template/runtime/Cargo.toml

```diff
@@ -49,7 +49,7 @@ fp-self-contained = { workspace = true }
 pallet-base-fee = { workspace = true }
 pallet-dynamic-fee = { workspace = true }
 pallet-ethereum = { workspace = true }
-pallet-evm = { workspace = true }
+pallet-evm = { workspace = true, features = ["forbid-evm-reentrancy"] }
```

Comment:
🎉

Reply:
We should make a test for this.  Let's create a separate ticket...

Reply:
https://github.com/GoldenGateGGX/golden-gate/issues/10